CRM/CloseAccountingPeriod/Form/PriorFinancialPeriod.php

```diff
@@ -61,9 +68,9 @@ public function setDefaultValues() {
   public function buildQuickForm() {
     $priorFinancialPeriod = CRM_CloseAccountingPeriod_BAO_CloseAccountingPeriod::getPriorFinancialPeriod($this->_contactID);
     if ($priorFinancialPeriod) {
-      $this->assign('priorFinancialPeriod', $priorFinancialPeriod);
+      $this->assign('priorFinancialPeriod', $priorFinancialPeriod['M'] . '-' . $priorFinancialPeriod['Y']);
```

Comment:
Joe, this is not for display. The value of the date is just being assigned to a variable to be used in smarty so that we can check if the user has changed the date on the form.